graph/src/env/mod.rs

```diff
@@ -311,6 +319,10 @@ impl EnvVars {
             genesis_validation_enabled: inner.genesis_validation_enabled.0,
             genesis_validation_timeout: Duration::from_secs(inner.genesis_validation_timeout),
             graphman_server_auth_token: inner.graphman_server_auth_token,
+            firehose_require_extended_blocks_for_chains:
+                Self::firehose_require_extended_blocks_for_chains(
```

Comment:
I think this should be the opposite, it should IMO require extended by default with the flag to authorize "downgrading" since the default option should be the safest option possible